code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -25,18 +25,25 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	var/tmp/temperature_archived = 0
 	var/volume = CELL_VOLUME //liters
 	var/last_share = 0
-	/// The fire key contains information that might determine the volume of hotspots.
-	var/list/reaction_results
+	/** 
+	  * List containing information about past reactions that have occured in this gas mixture.
+	  * 1: Numerical fire value 
+	  * 2: Assoc reaction list
+	  * 3: Misc list containing strings about the reaction
+	  */
+	var/list/reaction_results = list(0,list(),list())
```

Comment:
Changing this into assoc if that's fine

Reply:
I'll kick the misc index (3), it can be moved to the atom.
I'll make reaction_results a fully path-moles assoc then, hope this is fine.
Then change linda_fire to count the fire volume thing there locally instead of here.
Sounds sensible?

Reply:
:trollface: 
![image](https://user-images.githubusercontent.com/54709710/139542290-d87a7ced-e460-4a2f-8887-6de1e0dd58d3.png)

Reply:
> 
> 
> I'll kick the misc index (3), it can be moved to the atom. I'll make number two the whole list then, make reaction_results a fully path-moles assoc. Hope this is fine. Then change linda_fire to count the fire volume thing there locally instead of using the first index. Sounds sensible?

Why not just make the recording thing a text indexed assoc entry like fire? that way we don't need to have two entries by default for no reason

Reply:
Yep! Only one single "type" of entry here, everything is reaction_path - moles.